commit 55b1e7739a4efd2dcec9225baaa5ddba0e17fd0d
Author: Marcus Behrendt <marcus.behrendt.86@gmail.com>
Date:   Tue Dec 5 17:17:29 2023 +0100

    feat(container-creation-page): Add `privileged` option
    
    Fixes #752

diff --git a/src/view/container_creation_page.rs b/src/view/container_creation_page.rs
index 592cff6b..8979f19c 100644
--- a/src/view/container_creation_page.rs
+++ b/src/view/container_creation_page.rs
@@ -68,6 +68,8 @@ mod imp {
         #[template_child]
         pub(super) terminal_switch_row: TemplateChild<adw::SwitchRow>,
         #[template_child]
+        pub(super) privileged_switch_row: TemplateChild<adw::SwitchRow>,
+        #[template_child]
         pub(super) memory_switch: TemplateChild<gtk::Switch>,
         #[template_child]
         pub(super) mem_value: TemplateChild<gtk::Adjustment>,
@@ -543,6 +545,7 @@ impl ContainerCreationPage {
             .name(imp.name_entry_row.text().as_str())
             .pod(self.pod().as_ref().map(model::Pod::name))
             .terminal(imp.terminal_switch_row.is_active())
+            .privileged(imp.privileged_switch_row.is_active())
             .portmappings(
                 imp.port_mappings()
                     .iter::<model::PortMapping>()
diff --git a/src/view/container_creation_page.ui b/src/view/container_creation_page.ui
index 7d94a743..218439ec 100644
--- a/src/view/container_creation_page.ui
+++ b/src/view/container_creation_page.ui
@@ -174,6 +174,13 @@
                                           </object>
                                         </child>
 
+                                        <child>
+                                          <object class="AdwSwitchRow" id="privileged_switch_row">
+                                            <property name="title" translatable="yes">Privileged</property>
+                                            <property name="subtitle" translatable="yes">Give extended privileges to container</property>
+                                          </object>
+                                        </child>
+
                                         <child>
                                           <object class="AdwExpanderRow">
                                             <property name="enable-expansion" bind-source="memory_switch" bind-property="active" bind-flags="sync-create"/>
